pkg/installer/service/helper/main.go

```diff
@@ -254,11 +177,29 @@ func executeCommand() error {
 }
 
 func main() {
+	err := setupPriviledges()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error setting up priviledges: %s\n", err)
+		os.Exit(1)
+	}
 	log.SetOutput(os.Stdout)
-	err := executeCommand()
+	err = executeCommand()
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "error: %s\n", err)
 		os.Exit(1)
 	}
 	os.Exit(0)
 }
+
+func setupPriviledges() error {
+	if os.Getuid() != 0 {
+		return fmt.Errorf("only root can execute this command")
+	}
+	if err := syscall.Setuid(os.Getuid()); err != nil {
```

Comment:
Is this required? If I understand this correctly you already checked that UID is 0, and this seems to be setting it to 0 again (I might not understand seteuid enough)

Reply:
yes, it's an unfortunate hack required for subprocesses